internal/action/actions.go

```diff
@@ -2064,22 +2064,25 @@ func (h *BufPane) MouseMultiCursor(e *tcell.EventMouse) bool {
 	return true
 }
 
-// SkipMultiCursor moves the current multiple cursor to the next available position
-func (h *BufPane) SkipMultiCursor() bool {
+// SkipMultiCursor moves the current multiple cursor to the next or previous available position, depending on the argument `forward`.
```

Comment:
`SkipMultiCursor` still capitalized in this comment.

I think we don't need a comment here. It's an internal function, and from its usage below it is obvious what it does.

Reply:
Alright.